app/views/govuk_publishing_components/components/docs/button.yml

```diff
@@ -0,0 +1,78 @@
+name: Button
+description: Use buttons to move though a transaction, aim to use only one button per page.
+body: |
+  Button text should be short and describe the action the button performs.
+
+  [GOV.UK Elements has more information](https://govuk-elements.herokuapp.com/buttons/) on how buttons should be used.
+
+  Note: We do not consume GOV.UK Elements directly due to the naming conventions being leaky,
+  in time this component will be a wrapper for the [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend) project's button component.
+
+  Global button styles seen at `app/assets/stylesheets/helpers/_buttons.scss`
+  should be considered deprecated with this component replacing any instances.
+
+  govuk-* namespace replaced with pub-c-* otherwise we will conflict with [GOV.UK Frontend's namespace](https://github.com/alphagov/govuk-frontend/blob/master/docs/components.md)
```

Comment:
This detailed description seems like it may make more sense to be removed as I suspect it's detailed in this repository generally?